Lib/csv.py

```diff
@@ -403,6 +405,8 @@ def has_header(self, sample):
 
         columns = len(header)
         columnTypes = {}
+        average_size = 0
+        col_are_strings = True
         for i in range(columns): columnTypes[i] = None
```

Comment:
```suggestion
        columnTypes = {i: None for i in range(columns)}
        average_size = 0
        col_are_strings = True
```

Maybe going a bit overboard, but since this line was touched anyway, maybe worth using a comprehension here (I'm guessing this was probably written before they existed).